src/colorizer/ColorizeCanvas.ts

```diff
@@ -164,7 +258,11 @@ export default class ColorizeCanvas {
     if (!frame) {
       return;
     }
+    // Save frame resolution for later calculation
+    this.frameResolution = new Vector2(frame.image.width, frame.image.height);
```

Comment:
This feels like it should be the dataset's responsibility to store, not the canvas's... but there's not a satisfying way to derive it since we can't guarantee that all images in a dataset are the same size (though they absolutely should be). So I suppose it's better here.

Reply:
I would be comfy with building in the assumption that all frames of a dataset are same xy size.   Can't imagine why anyone would want otherwise...?   We could even put the values in our top level json file - we were also discussing adding a downsampling factor to resolve any possible differences between coordinates of centroid data vs size of images.

Reply:
Agreed that the dataset should hold this info! I'd like for centroid coordinates to match that of the image sizes, so that transformation would only need to be handled by the data generation script(s). Having this data in the manifest would be nice because then we don't have to wait for the first frame to be loaded to get its resolution... but I could also just grab the texture size for now.